manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -103,6 +103,8 @@ pdb:
   controllerManager:
     minAvailable: 1
 service: {}
+  # # Add health port to allow healthcheck probes to webhook service
```

Comment:
It looks like this is breaking our tests, since it gets clobbered by the autogen pipeline. IMO it's okay to remove these lines.